.github/workflows/integration-tests-reusable.yml

```diff
@@ -87,6 +87,22 @@ jobs:
       with:
         java-version: 18
         distribution: corretto
+    - name: Set up newest supported Go
+      if: matrix.target-language-version == 'newest'
```

Comment:
Just FYI, we aren't currently ever populating `matrix.target-language-version`, because running the entire suite twice for the sake of a small percentage of tests wasn't frugal. See https://github.com/dafny-lang/dafny/issues/1983 for more background, especially my latest comment on how I think we should solve this problem.

Reply:
This might be the reason the CI is still failing for you in fact.